fedimint-cli/src/lib.rs

```diff
@@ -241,7 +246,16 @@ impl Opts {
         api_secret: &Option<String>,
     ) -> CliResult<DynGlobalApi> {
         let our_id = self.our_id.ok_or_cli_msg("Admin client needs our-id set")?;
-        Ok(DynGlobalApi::from_config_admin(cfg, api_secret, our_id))
+
+        let connector = if self.use_tor {
+            Connector::Tor
+        } else {
+            Connector::default()
```

Comment:
maybe tor should be function as well `Connector::tor()`

Reply:
Sure, I'll use that instead